### PR TITLE
Fix 0.2.0 CMake Version and Change Log

### PR DESCRIPTION
**Pull request by sslattery**
Updates the CMake version and adds updates to the change log.

### CODE REVIEW

**Comment by dalg24** [comment]
You want to merge these into master.  But what's the plan about release 0.2?  Are you going to update the tag?

**Comment by sslattery** [comment]
Yes we would update the tag with these changes

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## [0.2.0]()
```
Comment (dalg24) [inline CHANGELOG.md]: What the deal with this syntax?  This is essentially an empty (invalid) URL.

Comment (sslattery) [reply CHANGELOG.md]: yeah that is weird. I have no idea why I did that. maybe it should link to the GitHub release page

**Comment by junghans** [comment]
I would make it v0.2.1, updating public tag is just bad practice.

File: CHANGELOG.md (hunk 1 above)
Comment (sslattery) [reply CHANGELOG.md]: fixed

**Comment by dalg24** [comment]
Answering here to https://github.com/ECP-copa/Cabana/issues/104#issuecomment-475797916 

@sslattery said:
> What about #85?

This PR was trying to fix:
* the runtime `Cabana::get_version()` that returns `"0.1.0"` but there `Cabana::git_commit_hash()` does the right thing
* the version number displayed on Doxygen but who's going to look at the 0.2.0 documentation after we release 0.3.0 which should be coming soon enough

Also this PR was done against master which was probably not the best unless we acted before a bunch other stuff got merged.  I suggest we forget about fixing 0.2.0 rebase #85 onto the current master, drop the to the version number change and cherry-pick the updates of the change log.  Then force-push and edit the subject line of the PR.

**Comment by dalg24** [comment]
I can take care of it if we agree on that.

**Comment by sslattery** [comment]
@dalg24 I agree with that strategy.

**Pull request closed without merging**